src/main/java/me/jellysquid/mods/sodium/mixin/features/debug/MixinDebugHud.java

```diff
@@ -35,6 +36,9 @@ private void appendRightText(CallbackInfoReturnable<List<String>> cir) {
             }
         }
 
+        strings.add("");
+        strings.add("Sodium Version: " + SodiumClientMod.version());
+
         strings.add("");
```

Comment:
I think it would make sense to remove the spacing here, so the two things added by Sodium are grouped